packages/gatsby-source-wordpress/src/scripts/download-test-data.js

```diff
@@ -0,0 +1,33 @@
+const fs = require(`fs`)
+const fetchData = require(`../fetch`)
+
+// Fetch data from our sample site and save it to disk.
+
+const typePrefix = `wordpress__`
+const refactoredEntityTypes = {
+  post: `${typePrefix}POST`,
+  page: `${typePrefix}PAGE`,
+  tag: `${typePrefix}TAG`,
+  category: `${typePrefix}CATEGORY`,
+}
+
+fetchData({
+  _verbose: false,
+  _siteURL: `http://dev-gatbsyjswp.pantheonsite.io`,
+  baseUrl: `dev-gatbsyjswp.pantheonsite.io`,
+  _useACF: true,
+  _hostingWPCOM: false,
+  auth: {
```

Comment:
You won't need to auth if you're hosted on Pantheon.io

Reply:
👍